docstr_coverage/config_file.py

```diff
@@ -0,0 +1,43 @@
+"""This module is for conf file .docstr.yaml"""
+import os
+
+import yaml
+
+
+def set_config_defaults(ctx, param, value):
+    """Update CLI option defaults in `ctx` to config file values
+
+    Parameters
+    ----------
+    ctx: click.Context
+        Click Context object
+    param: click.Parameter
+        Click Parameter object (assumed to be `config`)
+    value: String
+        Path to the configuration file
+
+    Returns
+    -------
+    String
+        Path to the configuration file"""
+    if (value is not None) and os.path.exists(value):
+        with open(value) as f:
+            config_data = yaml.safe_load(f) or {}
+
+        try:
+            # Check if `paths` were given in config file
+            config_paths = config_data.pop("paths")
+        except KeyError:
+            pass
+        else:
+            # Use config default if `paths` was not provided as CLI argument
+            if (not ctx.params.get("paths")) and config_paths:
```

Comment:
```suggestion
            if not ctx.params.get("paths") and config_paths:
```

I do not think we need the outer brackets. [Python Operator Precedence](https://www.mathcs.emory.edu/~valerie/courses/fall10/155/resources/op_precedence.html)
But maybe I'm missing something...